src/components/input/Input.tsx

```diff
@@ -164,19 +214,35 @@ export const Input: FunctionComponent<
         {type === "search" && loading && <Spinner />}
         <StyledInput
           type={type}
+          onClick={onClick}
           popup={popup}
           small={small}
           ref={ref}
           {...props}
         />
-        {status && (
-          <Icon
-            image={`INPUT_${status.toUpperCase()}`}
-            width="1.2rem"
-            height="1.2rem"
-          />
+        <IconsWrap>
+          {showClearButton && (
+            <Clear
+              onClick={() => onClear && onClear()}
```

Comment:
Maybe assign onClear a default value (e. g. function that returns undefined) instead of creating an inline function here that does the check.